Pokedex/AppDelegate.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  AppDelegate.swift
+//  Pokedex
+//
+//  Created by SAMEER SURESH on 9/25/16.
+//  Copyright © 2016 trainingprogram. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        
+        UINavigationBar.appearance().titleTextAttributes = [NSFontAttributeName: UIFont(name: "Pokemon-Pixel-Font", size: 24)!]
```

Comment:
I would suggest creating a file for the navigation controller with this line in it next time. You can then address the navigation bar of that specific navigation controller, and allow for the code to be grouped more intuitively.